src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
+import { Comment } from '../../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+export interface CommentInfoProps {
+  comment: Comment;
+}
+
+export const CommentInfo: React.FC<CommentInfoProps> = ({ comment }) => {
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">
+          {comment.name}
```

Comment:
It's not in the checklist for this assignment, but I'd recommend destructuring the properties

Reply:
That's right